WorkloadManagementSystem/Service/JobMonitoringHandler.py

```diff
@@ -289,16 +286,6 @@ def export_getJobLoggingInfo(jobID):
 
     return gJobLoggingDB.getJobLoggingInfo(jobID)
 
-##############################################################################
-  types_getJobsParameters = [list, list]
-
-  @staticmethod
-  @deprecated("Unused")
-  def export_getJobsParameters(jobIDs, parameters):
```

Comment:
In fact we are using this method in iLCDirac

PS: having the deprecated on the service side is not very useful, unless one reads the whole log...
```
NOTE: export_getJobsParameters function is deprecated (Unused).
```